src/code/z_lib.c

```diff
@@ -1,4 +1,12 @@
-#include "global.h"
+#include "z64lib.h"
+
+#include "main.h"
+#include "ichain.h"
+#include "sfx.h"
+#include "z64actor.h"
+#include "z64game.h"
+#include "functions.h"
+#include "macros.h" // for SQ
```

Comment:
```suggestion
#include "macros.h"
```
Are these comments necessary?